service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -285,15 +293,29 @@ private void sendNotification(String templateId, String email, Map<String, Objec
         }
     }
 
-    private void sendOrderNotificationForLocalAuthority(final CaseDetails caseDetails, final String localAuthorityCode,
-                                                        final String mostRecentUploadedDocumentUrl) {
+    private void sendGeneratedOrderNotificationToLocalAuthority(final CaseDetails caseDetails,
+                                                                final String localAuthorityCode,
+                                                                final String mostRecentUploadedDocumentUrl) {
         Map<String, Object> localAuthorityParameters =
-            orderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
+            generatedOrderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
                 caseDetails, localAuthorityCode, mostRecentUploadedDocumentUrl);
 
         String recipientEmail = inboxLookupService.getNotificationRecipientEmail(caseDetails, localAuthorityCode);
 
-        sendNotification(ORDER_NOTIFICATION_TEMPLATE, recipientEmail, localAuthorityParameters,
+        sendNotification(ORDER_NOTIFICATION_TEMPLATE_FOR_LA, recipientEmail, localAuthorityParameters,
+            Long.toString(caseDetails.getId()));
+    }
+
+    private void sendOrderIssuedNotificationToHmctsAdmin(final CaseDetails caseDetails,
+                                                         final String localAuthorityCode,
+                                                         final byte[] documentContents) {
+        Map<String, Object> hmctsParameters =
+            orderIssuedEmailContentProvider.buildOrderNotificationParametersForHmctsAdmin(
+                caseDetails, localAuthorityCode, documentContents);
+
+        String hmctsEmail = hmctsCourtLookupConfiguration.getCourt(localAuthorityCode).getEmail();
+
+        sendNotification(ORDER_NOTIFICATION_TEMPLATE_FOR_ADMIN, hmctsEmail, hmctsParameters,
```

Comment:
I would just use, parameters, and email. Don't think hmcts is needed

Reply:
👍